Bottom: b0ecaac03a4cee3fc35723b2693d5f56a0be1c5b
Top:    313ca82105b6466b196b9314bc12e16a2db1a299
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-22 21:50:00 +0000

hid/gtk: (GL) Disable double buffering of the GTK drawing widgets

 We already double buffer in GL, so we don't need GTK to "help" us here.
This might speed rendering up a little?


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index dad2d61..ed739b4 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -807,6 +807,8 @@ ghid_init_drawing_widget (GtkWidget *widget, GHidPort *port)
 {
   render_priv *priv = port->render_priv;
 
+  GTK_WIDGET_UNSET_FLAGS (GTK_WIDGET (widget), GTK_DOUBLE_BUFFERED);
+
   gtk_widget_set_gl_capability (widget,
                                 priv->glconfig,
                                 NULL,
